### PR TITLE
Sync table oids rake task

### PR DESCRIPTION
**Pull request by gfiorav**
Should aid in removing some problems where there is non-manipulatable table shown in a dashboard that is not fixed by ghost tables since it has a physical table but it's oid is not the UserTable table_id.

`bundle exec rake cartodb:tables:maintenance:sync_user_table_oids[<username>]`

Please CR, @juanignaciosl

### CODE REVIEW

File: lib/tasks/table_maintenance.rake [hunk 1]
```diff
@@ -0,0 +1,61 @@
+namespace :cartodb do
+  namespace :tables do
+    namespace :maintenance do
+      def ask_for_confirmation(key = 'continue')
```
Comment (juanignaciosl) [inline lib/tasks/table_maintenance.rake]: Don't do it now, but we need some kind of `RakeUtils` :_)

File: lib/tasks/table_maintenance.rake [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        raise 'A username must be provided' unless args[:username].present?
+
+        user = ::User.where(username: args[:username]).first
+
+        raise "No user with username '#{args[:username]}' found" if user.nil?
+
+        user_tables = user.tables.all
+
+        unsynced_user_tables = user_tables.reject do |user_table|
+          user_table.table_id != user_table.service.fetch_table_id
```
Comment (juanignaciosl) [inline lib/tasks/table_maintenance.rake]: Maybe this check is useful at `UserTable`, isn't there something similar at ghost tables logic? Feel free to ignore this comment though.

**Comment by juanignaciosl** [comment]
LGTM :+1:

File: lib/tasks/table_maintenance.rake (hunk 1 above)
Comment (gfiorav) [inline lib/tasks/table_maintenance.rake]: Totally agree :)

File: lib/tasks/table_maintenance.rake (hunk 2 above)
Comment (gfiorav) [inline lib/tasks/table_maintenance.rake]: I've thought of adding this there

**Pull request merged**